### PR TITLE
[CS-2373]: Get balances from oracle for tokens w/o coingekoid

### PR DESCRIPTION
**Pull request by danibonilha**
### Description

This is kind of a nightmare because the assets structure changes on multiple places, but this was the "better" way to handle this case, and hopefully we can change this data structure to have one source of truth. Also I'm handling the case the price is zero on the chart, so it won't show any value at the top

### Screenshots

Screenshot 

Screenshot

### CODE REVIEW

**Comment by eshinkawa** [comment]
looks good overall with some minor observations, did not approve because CI is failing

File: cardstack/src/helpers/fallbackExplorerHelper.ts [hunk 1]
```diff
@@ -9,11 +9,13 @@ import { isNativeToken } from '@cardstack/utils/cardpay-utils';
 import {
   AssetType,
   AssetWithNativeType,
+  CurrencyConversionRates,
   DepotType,
   PrepaidCardType,
   TokenType,
 } from '@cardstack/types';
 import { Network } from '@rainbow-me/helpers/networkTypes';
+import { getNativeBalance } from '@cardstack/services';
```
Comment (topwebtek7) [inline cardstack/src/helpers/fallbackExplorerHelper.ts]: can we change this service name `getNativeBalance` to be more make sense like `getNativeBalanceFromOracle`, as there are similar names cardstack utils as well.

**Review by topwebtek7 — COMMENTED** [review]
Looks good to me and left just a comment

**Review by topwebtek7 — APPROVED** [review]

File: cardstack/src/helpers/fallbackExplorerHelper.ts (hunk 1 above)
Comment (topwebtek7) [reply cardstack/src/helpers/fallbackExplorerHelper.ts]: image

we need to update depo send sheet UI for tokens without balances as has too much space but I think we can fix it in a separate ticket

Comment (danibonilha) [reply cardstack/src/helpers/fallbackExplorerHelper.ts]: Agreed, and it's a different issue, since it's totally related to UI 👍

**Pull request merged**